server/data/index.ts

```diff
@@ -0,0 +1,3 @@
+import cycles from './cycles';
```

Comment:
I had a different vision for this based on how index files typically work, see my pull request for user data/routes for how I use it.